src/components/Error/Error.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { FC } from 'react';
+
+type Props = {
+  hasError: string,
+  setHasError: (error: string) => void,
+};
+
+export const Error: FC<Props> = ({ hasError, setHasError }) => {
+  return (
+    <>
+      {hasError && (
+        <div className="
+          notification is-danger
+          is-light has-text-weight-normal
+          "
+        >
+          <button
+            type="button"
+            className="delete"
+            onClick={() => setHasError('')}
+          />
+          {hasError}
+        </div>
+      )}
+      ;
```

Comment:
There is an unnecessary semicolon (;) at the end of line 26. Please remove it to keep the code clean. 😊